graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/ElasticsearchClient.java

```diff
@@ -86,6 +90,16 @@ private SearchResponse firstResponseFrom(MultiSearchResponse result, String erro
         }
     }
 
+    public <R> R executeUnsafe(ThrowingBiFunction<RestHighLevelClient, RequestOptions, R, IOException> fn, String errorMessage) throws IOException {
```

Comment:
As discussed with @dennisoelkers, this method should get a more precise name (and possibly an explanatory comment - maybe even be marked as `@Deprecated`) to indicate that it's only necessary to be compatible with existing handling of `IOException`s in scrolling-related code.

Reply:
:heavy_check_mark: